rust-runtime/aws-smithy-compression/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "aws-smithy-compression"
+version = "0.0.1"
+authors = [
+  "AWS Rust SDK Team <aws-sdk-rust@amazon.com>",
+  "Zelda Hessler <zhessler@amazon.com>",
```

Comment:
The defacto process so far has been using the name of the actual person who created a crate as well as the team name.

We can change it if you like. It might hurt my vanity a bit though 😆

Reply:
I don't care I was more asking if there was some additional rationale I'm missing. Keep your vanity 😆